.github/workflows/deploy_docs.yml

```diff
@@ -0,0 +1,43 @@
+name: Deploy docs
+
+on:
+  push:
+    branches: [ 386-Build-TriBITS-Documentation-with-Sphinx-and-ReadTheDocs ]
+
+jobs:
+  build-and-deploy-docs:
+
+    runs-on: ubuntu-latest
+    env:
+      working-directory: ./tribits/doc/sphinx
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 2.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: '2.7'
+          architecture: 'x64'
```

Comment:
Boy, we need to get rid of this requirement to install Python 2.7.  That should **not** be necessary.   This must have to do with the `python` name hack.  Note that the underlying tool [extract_rst_cmake_doc.py](https://github.com/TriBITSPub/TriBITS/blob/master/tribits/python_utils/extract_rst_cmake_doc.py) runs just fine with Python 3.x as shown by:

```
[rabartl@crf450 TRIBITS_MPI_DEBUG_PYTHON_3 (crf450)]$ which python
~/bin/python-3.5.2/python
[rabartl@crf450 TRIBITS_MPI_DEBUG_PYTHON_3 (crf450)]$ python --version
Python 3.5.2
[rabartl@crf450 TRIBITS_MPI_DEBUG_PYTHON_3 (crf450)]$ ctest -R extract_rst_cmake_doc
Test project /ascldap/users/rabartl/Trilinos.base/BUILDS/GCC-4.8.3/TRIBITS_MPI_DEBUG_PYTHON_3
    Start 242: TriBITS_extract_rst_cmake_doc_UnitTests
1/7 Test #242: TriBITS_extract_rst_cmake_doc_UnitTests ..............................   Passed    0.15 sec
    Start 243: TriBITS_extract_rst_cmake_doc_help
2/7 Test #243: TriBITS_extract_rst_cmake_doc_help ...................................   Passed    0.08 sec
    Start 244: TriBITS_extract_rst_cmake_doc_extract_1_block_1_file
3/7 Test #244: TriBITS_extract_rst_cmake_doc_extract_1_block_1_file .................   Passed    0.07 sec
    Start 245: TriBITS_extract_rst_cmake_doc_extract_2_blocks_2_files
4/7 Test #245: TriBITS_extract_rst_cmake_doc_extract_2_blocks_2_files ...............   Passed    0.06 sec
    Start 246: TriBITS_extract_rst_cmake_doc_extract_2_blocks_glob_dir
5/7 Test #246: TriBITS_extract_rst_cmake_doc_extract_2_blocks_glob_dir ..............   Passed    0.12 sec
    Start 247: TriBITS_extract_rst_cmake_doc_extract_and_replace_2_blocks
6/7 Test #247: TriBITS_extract_rst_cmake_doc_extract_and_replace_2_blocks ...........   Passed    0.15 sec
    Start 248: TriBITS_extract_rst_cmake_doc_extract_and_replace_2_blocks_notrace
7/7 Test #248: TriBITS_extract_rst_cmake_doc_extract_and_replace_2_blocks_notrace ...   Passed    0.15 sec

100% tests passed, 0 tests failed out of 7

Subproject Time Summary:
TriBITS    =   0.78 sec*proc (7 tests)

Total Test time (real) =   0.85 sec
```

Therefore, I think we need to make the driver script a little smarter and run:

```
$ <python-exec>  extract_rst_cmake_doc.py ...
```

instead of:

```

```
$ extract_rst_cmake_doc.py ...
```

which relies on `python` to be in the path.

I bet this is a really easy fix.